release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -92,6 +92,16 @@ For more information about enabling a feature set during installation, see xref:
 ==== {product-title} on ARM
 {product-title} {product-version} is now supported on ARM architecture-based Azure installer-provisioned infrastructure. AWS Graviton 3 processors are now available for cluster deployments and are also supported on {product-title} 4.11. For more information about instance availability and installation documentation, see xref:../installing/installing-preparing.html#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms]
 
+[id="ocp-4-12-consistent-address-for-ironicapi"]
+==== Consistent IP address for Ironic API in bare-metal installations without a provisioning network  
+With this update, in bare-metal installations without a provisioning network, the Ironic API service is accessible through a proxy server. This proxy server provides a consistent IP address for the Ironic API service. If the Metal3 pod that contains `metal3-ironic` relocates to another pod, the consistent proxy address ensures constant communication with the Ironic API service.
+
+[id="ocp-4-12-ironic-container-image-rhel9"]
+==== Ironic container images use RHEL 9 base image
+In earlier versions of {product-title}, Ironic container images used Red Hat Enterprise Linux (RHEL) 8 as the base image. From {product-title} 4.12, Ironic container images use RHEL 9 as the base image. The RHEL 9 base image adds support for CentOS Stream 9, Python 3.8, and Python 3.9 in Ironic components.
```

Comment:
```suggestion
In earlier versions of {product-title}, Ironic container images used Red Hat Enterprise Linux (RHEL) 8 as the base image. Starting with {product-title} 4.12, Ironic container images use RHEL 9 as the base image. The RHEL 9 base image adds support for CentOS Stream 9, Python 3.8, and Python 3.9 in Ironic components.
```

Reply:
Since 4.12 is the new release I think this seems more fitting verbiage. Small nit.